src/algebra/indicator_function.lean

```diff
@@ -249,6 +249,18 @@ begin
   rw [pi.mul_apply, this, one_mul]
 end
 
+@[to_additive] lemma mul_indicator_mul_compl_eq_piecewise (f g : α → M) :
```

Comment:
I think this is missing a decidable argument

Reply:
It compiles for me so I'm not sure what decidable argument is needed here

Reply:
The RHS of this lemma is about `classical.dec`, but should be about any decidable argument

Reply:
do you want me to add `[∀ x, decidable (x ∈ s)]`?

Reply:
Yes, but with spelling `[decidable_pred (∈ s)]